.eslintrc.yml

```diff
@@ -1,5 +1,15 @@
 ---
 root: true
+env:
+  mocha: true
 extends: standard
+plugins:
+  - mocha
+  - chai-friendly
 rules:
-  'no-console': error
+  "no-console": error
+  "mocha/no-exclusive-tests": error
+  "mocha/no-skipped-tests": error
+  "mocha/no-identical-title": error
+  "no-unused-expressions": off
+  "chai-friendly/no-unused-expressions": error
```

Comment:
while we are at linting code can we requires trailing commas ? https://eslint.org/docs/rules/comma-dangle

Reply:
As discussed, I'd like not to change the linting rules until I'm fully familiar with the project